styleguide/components/Nav.css

```diff
@@ -1,5 +1,4 @@
 .nav {
-  composes: root from '../../components/Container/Container.css';
```

Comment:
I assume this broke because it references Container.css which is no longer imported..?

https://github.com/PactCoffee/loggins/pull/35/files#diff-900bafaaf222fc9937c1135eeb27c6aeL3